pontis-ui/src/components/Landing/ProvidersSection.tsx

```diff
@@ -12,12 +12,12 @@ const providers = [
 ];
```

Comment:
Could change this to have full image name (`ftx.svg`) in case not all of them are svgs at some point

Reply:
Seems like a good idea, unless you use this array elsewhere for purposes other than mapping to svg names. If not and you add ".svg", would also rename the array